.doctrine-project.json

```diff
@@ -5,21 +5,27 @@
     "docsSlug": "doctrine-coding-standard",
     "versions": [
         {
-            "name": "6.0",
+            "name": "7.0",
             "branchName": "master",
             "slug": "latest",
             "upcoming": true
         },
         {
-            "name": "5.0",
-            "branchName": "5.x",
-            "slug": "5.0",
+            "name": "6.0",
+            "branchName": "6.x",
```

Comment:
I noticed that we don't have the `6.x` branch. Should we create it?

Reply:
Only after release, not yet.